tests/Tests/100__deploy/100__installation/109__operator.robot

```diff
@@ -47,6 +47,60 @@ Verify That The Operator Pod Does Not Get Stuck After Upgrade
             Log Error And Fail Pods When Pods Were Terminated    ${operator_pod_info}    Opertator Pod Stuck
         END
     END
+Verify Clean Up ODS Deployer Post-Migration
+    [Tags]    Tier1
+    ...       ODS-1767
+    ${version_check} =    Is RHODS Version Greater Or Equal Than    1.17.0
+    IF    ${version_check} == False
+        Log    Skipping test case as RHODS version is less than 1.17.0
+        Skip
+    END
```

Comment:
There is a Keyword which can  reduce the number of line  to single one . `Skip If RHODS Version Greater Or Equal Than    version=1.17.0`

Reply:
In this case it should skip if version < 1.17.

@pablofelix you could create a keyword `Skip If RHODS Version Less Or Equal Than` in tests/Resources/Common.robot ?

Reply:
yes correct